src/Aquarius.Client/Helpers/SdkServiceClient.cs

```diff
@@ -1,12 +1,25 @@
-﻿using ServiceStack;
+﻿using System;
+using ServiceStack;
 
 namespace Aquarius.Helpers
 {
     public class SdkServiceClient : JsonServiceClient
     {
+        public static TimeSpan? FirstByteReceivedTimeout { get; set; }
+        public static TimeSpan? ReadEntireResponseTimeout { get; set; }
+
+        static SdkServiceClient()
+        {
+            FirstByteReceivedTimeout = AppSettings.Get<TimeSpan?>(nameof(FirstByteReceivedTimeout), TimeSpan.FromMinutes(3));
+            ReadEntireResponseTimeout = AppSettings.Get<TimeSpan?>(nameof(ReadEntireResponseTimeout), TimeSpan.FromMinutes(4));
```

Comment:
@DougSchmidt-AI This was originally 5 minutes and you've documented it as such. Is it wrong?

Reply:
Good catch. It's wrong (a silly typo from me) and should have been left at 5 minutes. I'll fix that.

I don't think anyone will be affected, since all the timeouts experienced when talking to our ServiceStack-based back ends will tend to hit the `FirstByteReceivedTimeout`.

The ServiceStack response pipeline has the effect of serializing the response DTO to JSON into the response stream. So the time it takes to send the response is essentially the time it takes to serialize JSON into the HTTP response stream. Usually that is under a second, even for a 10 MB JSON document.

I've never seen a response that sends its first response byte within the first timeout, but then slows down sending out the rest of the JSON response slow enough that the 5-minute timeout was triggerred.

Other non-ServiceStack back-ends might serialize their responses in chunks, and that's where Microsoft's `HttpWebRequest.ReadWriteTimeout` comes in handy.

But I should change the timeout back to 5-minutes to match the documented behaviour.

Reply:
Fixed by #82 and deployed to NuGet as 18.7.3